frontend/src/app/users/users-profile/users-profile.component.html

```diff
@@ -0,0 +1,27 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+<h2>Profile of {{ userName }}</h2>
+
+<div *ngIf="userId !== currentUser?.id">
+  Common Projects
```

Comment:
This is a small title and should be indicated as title, make it bold and a bit larger